src/query/src/error.rs

```diff
@@ -56,20 +56,6 @@ pub enum Error {
         location: Location,
     },
 
-    #[snafu(display("Catalog not found: {}", catalog))]
-    CatalogNotFound {
```

Comment:
Are those error not used before, or removed in this patch?

Reply:
They were previously used but got deprecated during refactoring, and it was an oversight not to remove them. I have a python script to find unused errors, so try to remove some at this patch.